front/pages/w/[wId]/u/gens/index.tsx

```diff
@@ -163,14 +163,17 @@ export class FunctionSingleArgStreamer {
   }
 
   feed(token: string): void {
+    // pseudo rule for parsing here because openAI makes mistakes
```

Comment:
I think this is too brittle since `{<SPACE><RETURN>` as an example would probably crash here

Reply:
I think that would be a nice trade-off here?

If the app update has no effect let's revert it?